pkg/proxy/proxy_integration_test.go

```diff
@@ -415,6 +430,41 @@ func TestIntegrationGetLoadtest(t *testing.T) {
 		assert.NotEqual(t, apisLoadTestV1.LoadTestErrored, dat.Phase)
 		assert.Equal(t, false, dat.HasTestData)
 	})
+
+	t.Run("Get loadtest details from gRPC/REST gateway", func(t *testing.T) {
+		req, err := http.NewRequest(http.MethodGet, fmt.Sprintf("http://localhost:%d/v2/load-test/%s", restPort, expectedLoadtestName), nil)
+		require.NoError(t, err, "Could not create GET request")
+
+		res, err := http.DefaultClient.Do(req)
+		require.NoError(t, err, "Could not send GET request")
+		require.Equal(t, http.StatusOK, res.StatusCode)
```

Comment:
Should this not be placed after the `defer` on line 442? or changed to be an `assert` call

Reply:
true, fixed